__tests__/src/reducers/contacts.js

```diff
@@ -0,0 +1,56 @@
+import reducer, { initialState } from '../../../src/reducers/contacts';
+import {
+  startContactsFetch,
+  contactsUpdated,
+  addContact,
+} from '../../../src/actions/contacts';
+import { servicesDestroyed } from '../../../src/actions/serviceContainer';
+
+describe('contacts reducer action handling', () => {
+  const mockAddress = '0xtestaddress';
+  const callbackMock = jest.fn();
+  const contactsMock = {
```

Comment:
I guess naming should be `contactMock` since it's not an array, but object.

Reply:
Done